compiler/rustc_mir_build/src/thir/mod.rs

```diff
@@ -11,4 +11,3 @@ crate mod cx;
 crate mod pattern;
 
 mod util;
-pub mod visit;
```

Comment:
Should the module file be deleted?

Reply:
didn't we just move that file to `rustc_middle`?

Reply:
Sorry I missed it.